apis/controllers/cartController.js

```diff
@@ -0,0 +1,44 @@
+const { cartService } = require('../services');
+const { catchAsync } = require('../utils/error');
+
+const getCart = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const result = await cartService.getCartByUserId(userId);
+
+  res.status(200).send({ result });
+});
+
+const postCart = catchAsync(async (req, res) => {
+  const userId = req.user.id;
+  const { productId, productOptionId, quantity } = req.body;
+
+  await cartService.postCart(productOptionId, quantity, userId);
+
+  res
+    .status(201)
+    .send({ message: `Product ${productId} was added to ${userId}'s cart` });
```

Comment:
message는 고정적이여야 합니다. 현재는 productID와 userID에 의해 계속 유동적으로 변합니다. 
response에서 특정 값들을 보고 싶다면 아래와 같이 키를 구분해 주세요.
```suggestion
    .send({ message: `Cart was created`, productId : productId, userId: userId });
```

Reply:
수정완료